backend/YouTubeV2.Application/Services/UserService.cs

```diff
@@ -77,6 +77,9 @@ public async Task RegisterAsync(RegisterDto registerDto, CancellationToken cance
 
         public ClaimsPrincipal? ValidateToken(string token) => _jwtHandler.ValidateToken(token);
 
+        public string? GetUserId(IEnumerable<Claim> claims) => JwtHandler.GetUserId(claims);
```

Comment:
This function will probably repeat in all controllers. Maybe create abstract class ourController which has this method and inherits from ControllerBase and our controllers can inherit from that

Reply:
I now realize that this is a service not a controller. Still i think my solution is better than passing user service to all controllers